The build exr logic is inverted, so un-invert it.

http://bugs.gentoo.org/95396
https://sourceforge.net/tracker/index.php?func=detail&aid=1216798&group_id=2997&atid=302997

--- acinclude.m4
+++ acinclude.m4
@@ -325,10 +325,10 @@
               [build_exr=$enableval],
               [build_exr=no])
 
-if test "x$build_exr" = "xno" ; then
+if test "x$build_exr" = "xyes" ; then
 	PKG_CHECK_MODULES(OPENEXR, OpenEXR, [build_exr=yes], [build_exr=no])
 
-	if test "x$build_exr" = "xno" ; then
+	if test "x$build_exr" = "xyes" ; then
 	   	AC_CONFIG_FILES([ PlugIns/EXRCodec/Makefile \
     					 PlugIns/EXRCodec/src/Makefile \
     					 PlugIns/EXRCodec/include/Makefile])
